### ISSUE TITLE
The checkbox "Enable Steam Play for all other titles" stays disabled after restart

### ISSUE THREAD

**Issue by MightyCreak** [flathub/com.valvesoftware.Steam.CompatibilityTool.Proton#32]
Hello!

I've installed Steam.CompatibilityTool.Proton (through GNOME Software), checked the checkbox "Enable Steam Play for all other titles" and selected "Proton 5.13-4 (community build 1608395899)".

Aft6er being prompted to restart Steam, I went back to the Steam settings and the checkbox "Enable Steam Play for all other titles" stays disabled.

The fact that it is disable is apparently not a problem to launch the games using Steam Play (or at least not a problem to run Doom Eternal), but it would be nice if the checkbox would stay checked.

See also: https://github.com/flathub/com.valvesoftware.Steam/issues/560#issuecomment-753367046

Cheers!

**Comment by gasinvein** [comment]
Yeah, this appears to be a Steam client issue ValveSoftware/steam-for-linux#6043 that affects all 3rd-party compatibility tools, previously reported in #6. Sadly we can't do anything about it from flatpak side.
